loops/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace loops
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // TaskOne();
+            // TaskTwo();
+            TaskThree(); 
+        }
+
+        static void TaskOne(){
+            Console.WriteLine("Pls input n: ");
+            int n = Int32.Parse(Console.ReadLine());
+            double result = 0;
+
+            for(double i = 1; i < n + 1; i++){
+                result += (i + 1) / i;
+                // Console.WriteLine(result);
+            }
+
+            Console.WriteLine(result);
+        }
+
+        static void TaskTwo(){
+            Console.WriteLine("Start writing numbers: "); 
+            int newNumber = Int32.Parse(Console.ReadLine());
+            int count = 0; 
+
+            while (newNumber != 0){
+                newNumber = Int32.Parse(Console.ReadLine()); 
+                count++; 
+            }
+
+            Console.WriteLine("total of {0} numbers", count);
+        }

```

Comment:
Задача сделана не полностью. Нужно подсчитать не только количество чисел, но и их сумму.
Лучше использовать зарезервированные в языке программирования слова, а не системные обозначения типов.
```suggestion
        static void TaskTwo(){
            Console.WriteLine("Start writing numbers: "); 
            int newNumber = int.Parse(Console.ReadLine());
            int count = 0, sum = 0; 

            while (newNumber != 0){
                newNumber = int.Parse(Console.ReadLine());
                sum += newNumber;
                count++; 
            }

            Console.WriteLine($"Total numbers: {count}. Sum: {sum}");
        }
```